Source/Windows/Common/InvalidationTracker.h

```diff
@@ -22,13 +22,17 @@ class InvalidationTracker {
   InvalidationTracker(FEXCore::Context::Context& CTX, const std::unordered_map<DWORD, FEXCore::Core::InternalThreadState*>& Threads);
   void HandleMemoryProtectionNotification(uint64_t Address, uint64_t Size, ULONG Prot);
   void HandleImageMap(uint64_t Address);
-  void InvalidateContainingSection(uint64_t Address, bool Free);
+  struct InvalidateContainingSectionResult {
```

Comment:
Wouldn't it make sense to use `IntervalList<uint64_t>::Interval` here instead of defining a new type that's only used once?

Reply:
I had it as that but thought to avoid confusion between size/end a struct made more sense